core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -135,7 +135,7 @@ class SearchService(
     // index all the given bases and return number of rows written
     def indexBases(bases: Set[FileObject], checks: Seq[FileCheck]): Future[Int] = {
       log.debug("Indexing bases...")
-      val checksLookup: Map[String, FileCheck] = checks.map(check => (check.filename -> check)).toMap
```

Comment:
you might want to back out some of these cleanups as it'll introduce conflicts with the graphpocalypse branch.